app/models/broadcast.rb

```diff
@@ -23,7 +23,7 @@ def self.receivable_association
   WHICH_PHONE_OPTIONS = %w(main_only alternate_only both)
 
   # options for recipients
-  RECIPIENT_SELECTION_OPTIONS = %w(all_users all_observers specific)
+  RECIPIENT_SELECTION_OPTIONS = %w(all_users all_enumerators specific)
```

Comment:
I think we will need a migration for this one, no?

Reply:
yes!